src/kite/base.js

```diff
@@ -10,27 +10,34 @@ import handleIncomingMessage from './handleIncomingMessage'
 import enableLogging from './enableLogging'
 import Timeout from './timeout'
 import KiteError from './error'
+import MessageScrubber from './messagescrubber'
 import { Event, AuthType, Defaults, TimerHandles, State } from '../constants'
 
-function __guard__(value, transform) {
-  return typeof value !== 'undefined' && value !== null
-    ? transform(value)
-    : undefined
-}
-
 class Kite extends Emitter {
-  constructor(options) {
+  static version = Defaults.KiteInfo.version
+  static Error = KiteError
+  static transportClass = WebSocket
+
+  static defaultOptions = {
```

Comment:
I guess this is to prevent re-assigning a value to a class constant, right? I am not sure if we really need this, because of the fact that we need to write down something 3 times 

```js
const constant1 = 33, // 1
      constant2 = 2;

class Example {

  static get constant1() { // 2
    return constant1; // 3
  }

  static get constant2() {
    return constant2;
  }
}
```
vs
```js
class Example {
  static constant1 = 33;
  static constant2 = 2;
}
```

If it's not related to constants being reassigned, then can you please explain the reason behind?